StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedFormViewController.swift

```diff
@@ -70,25 +70,29 @@ class EmbeddedFormViewController: UIViewController {
             navigationBar.isUserInteractionEnabled = isUserInteractionEnabled
         }
     }
+    
+    var collectsUserInput: Bool {
+        return paymentMethodFormViewController.form.collectsUserInput
+    }
 
     enum Error: Swift.Error {
         case noPaymentOptionOnBuyButtonTap
     }
     var selectedPaymentOption: PaymentSheet.PaymentOption? {
         return paymentMethodFormViewController.paymentOption
     }
-
-    private let loadResult: PaymentSheetLoader.LoadResult
+    
     private let paymentMethodType: PaymentSheet.PaymentMethodType
     private let configuration: EmbeddedPaymentElement.Configuration
     private let intent: Intent
     private let elementsSession: STPElementsSession
+    private let savedPaymentMethods: [STPPaymentMethod]?
```

Comment:
1. Seems like `savedPaymentMethods` can be non-optional
2. We technically don't even have to pass in the entire array. Instead we can either pass in:

* a boolean -- "shouldUseNewCardNewCardHeader: Bool" or
* `savedPaymentMethod.first` type: `STPPaymentMethod?`

Reply:
Ah, I like this better